src/client/components/Atoms/CurvedButton/CurvedButton.jsx

```diff
@@ -0,0 +1,24 @@
+import styles from './CurvedButton.module.css';
+
+export const CurvedButton = ({ title, type = 'button', greenOrBlack, onClick = () => {} }) => {
```

Comment:
Exports: sticking to the estabilished convention declare component as it is without directly exporting it.
Instead use last line to export button

f.ex.
`export { CurvedButton }`

just in case if we will export more than one variable from file

Reply:
It doesn't stops us from exporting more variables. Just in each new you add export const or function or whatever. And import is the same syntax. For example
` import { function1, variable1, ...etc} from 'pathToFile.js'`

Reply:
True but we will have then many export styles in one app.
Let's keep going with same standard everywhere.